example-model/src/main/java/uk/gov/gchq/palisade/example/configurator/ExampleUsers.java

```diff
@@ -14,9 +14,12 @@
  * limitations under the License.
  */
 
-package uk.gov.gchq.palisade.example.common;
+package uk.gov.gchq.palisade.example.configurator;
 
 import uk.gov.gchq.palisade.User;
+import uk.gov.gchq.palisade.example.common.ExampleUser;
+import uk.gov.gchq.palisade.example.common.Role;
+import uk.gov.gchq.palisade.example.common.TrainingCourse;
 
 public final class ExampleUsers {
 
```

Comment:
Raise a bug and added to next sprint to remove ExampleUsers from the example repo fully

Reply:
This has been done. PAL-613 added to the next sprint (sprint 20).

Reply:
https://jira.tdx.oneit.gov.uk/browse/PAL-613 raised